gate-plugins/src/main/kotlin/com/netflix/spinnaker/gate/plugins/deck/DeckPluginService.kt

```diff
@@ -78,9 +78,15 @@ class DeckPluginService(
       fun from(file: File): PluginAsset {
         return PluginAsset(
             contentType = when {
-              file.toString().endsWith(".js") -> { "application/javascript" }
-              file.toString().endsWith(".css") -> { "text/css" }
-              file.toString().endsWith(".html") -> { "text/html" }
+              file.toString().endsWith(".js") -> {
+                "application/javascript"
+              }
+              file.toString().endsWith(".css") -> {
+                "text/css"
+              }
+              file.toString().endsWith(".html") -> {
+                "text/html"
+              }
```

Comment:
Thanks.

Reply:
🤷‍♂ That was just something spotless did.

Reply:
Thanks spotless.